eftest/src/eftest/report/junit.clj

```diff
@@ -110,13 +110,18 @@
     (locking flush-lock (g) (f))
     (swap! *context* update ::deferred-report dissoc ns)))
 
+(defmethod report :begin-test-var [m]
+  (swap! *context* assoc-in [::test-start-times (:var m)] (System/nanoTime)))
+
 (defmethod report :end-test-var [m]
   (let [ns (-> (:var m) meta :ns ns-name name)
+        duration (- (System/nanoTime) (get-in @*context* [::test-start-times (:var m)]))
```

Comment:
Can you break this up into two lines to keep the line length under 80 characters?